components/molecules/TopicViewerContent.tsx

```diff
@@ -19,6 +20,11 @@ const useStyles = makeStyles((theme) => ({
     marginRight: theme.spacing(-3),
     marginBottom: theme.spacing(2),
     marginLeft: theme.spacing(-3),
+    backgroundColor: gray[800],
+    "& > *": {
+      maxWidth: "calc(40vh * 16 / 9)",
```

Comment:
16:9を前提にしたスタイルになっており、4:3など縦長のアスペクト比だとより高さをとってしまう

なぜmaxHeightではなくmaxWidthなのかというと、アスペクト比を維持してレスポンシブにするスタイルとして `width: 100%; padding-top: 56.25%;` のようなスタイルが当てられており、maxHeightでは高さの制限が出来ないため

Reply:
詳しい説明ありがとうございます!
PRのコメントは流れていくと思うのでひょっとするとコードにコメントとして記載していくほうが良いかなとも思いました。いかがでしょうか?

Reply:
動画プレイヤー(Vimeo、Video.js)のソースに応じた表示領域の変化について検証したところ、

- ソース16:9 → プレイヤーも16:9で表示
- ソース4:3 → プレイヤーも4:3で表示
- ソース1:2 → プレイヤーは16:9で表示

のような挙動だと分かったので、4:3を前提としたスタイルに変更するのが良いかと思った(プレイヤーが16:9で表示される場合は現在よりさらに小さく表示されることになる)

```suggestion
      maxWidth: "calc(40vh * 4 / 3)",
```

Reply:
> 動画プレイヤー(Vimeo、Video.js)のソースに応じた表示領域の変化について検証したところ、
> 
>     * ソース16:9 → プレイヤーも16:9で表示
> 
>     * ソース4:3 → プレイヤーも4:3で表示
> 
>     * ソース1:2 → プレイヤーは16:9で表示
> 
> 
> のような挙動だと分かったので、4:3を前提としたスタイルに変更するのが良いかと思った(プレイヤーが16:9で表示される場合は現在よりさらに小さく表示されることになる)

コメントを追加する際にこの変更もコミットしてしまったので、

- maxWidthを使っていることへのコメント
- 4:3前提であることへのコメント

を追加しました！